dist/profile/manifests/ssl-cert.pp

```diff
@@ -0,0 +1,29 @@
+#
+# SSL certificates
+#
+class profile::ssl-cert (
```

Comment:
Since this is a pretty Apache-specific profile, I'd rather it be named "apache-cert" or something like that to make sure the apache coupling is very very clear